enterprise/models.py

```diff
@@ -2111,6 +2125,13 @@ class LearnerCreditEnterpriseCourseEnrollment(EnterpriseFulfillmentSource):
     .. no_pii:
     """
 
+    def renew(self, transaction_id=None, **kwargs):
+        """
+        Idmpotently reactivates this LearnerCreditEnterpriseCourseEnrollment.
+        """
+        self.transaction_id = transaction_id
```

Comment:
It's probably worth throwing an exception if the transaction id hasn't changed from the original fulfillment to the renewed fulfillment.  Or at least log a warning, maybe?